src/power_grid_model_io/functions/_functions.py

```diff
@@ -129,12 +129,26 @@ def both_zeros_to_nan(value: float, other_value: float) -> float:
         return float("nan")
     return value
 
+
 def find_terminal_type(**kwargs) -> MeasuredTerminalType:
     """
     Return the measured terminal type, based on the string representation
-    """ 
+    """
     for key, id in kwargs.items():
         if id is not None:
             return MEASURED_TERMINAL_TYPE_MAP[key]
     _LOG.warning("No measured terminal type is found!")
-    return float("nan")
\ No newline at end of file
+    return float("nan")
+
+
+def if_not_link(input_string) -> bool:
+    """
+    Check if the measurement field is applied on a link
+    """
+    return input_string != "link"
+
+def filter_if_object(object_name: str, excl_object: str) -> bool:
```

Comment:
here and in other places: please consistently use 2 whitelines when there is no indentation, and 1 whiteline if there is indentation. e.g.since `def filter_if_object` is not indented, this should be prefixed with 2 whitelines
```suggestion
    return input_string != "link"


def filter_if_object(object_name: str, excl_object: str) -> bool:
```